java/integTest/src/test/java/glide/BatchTestUtilities.java

```diff
@@ -449,6 +488,46 @@ private static Object[] hashCommands(BaseBatch<?> batch, boolean isAtomic) {
                     }, // hscan(hashKey2, "0", HScanOptions.builder().count(20L).build());
                 };
 
+        // Add expected results for hash field expiration commands (Valkey 9.0.0+)
+        if (SERVER_VERSION.isGreaterThanOrEqualTo("9.0.0")) {
+            result =
+                    concatenateArrays(
+                            result,
+                            new Object[] {
+                                2L, // hsetex(hashKey3, Map.of(field1, value1, field2, value2), expiryOptions)
+                                new String[] {
```

Comment:
Probably (for all arrays)
```suggestion
                                new Object[] {
```
batch has no response processing, so all arrays are uncasted (arbitrary `Object[]`).
But if tests pass - feel free to keep as is.